app/models/user.rb

```diff
@@ -18,8 +18,11 @@ class CannotChangeRole < StandardError; end
 
   before_create :set_invitation_expires_at
 
+  # Allow data_analyst users to be deleted because they cannot
+  # be assigned another 'lesser' role - see AddDataAnalystRoleEnum
+  # migration which destroys data_analysts if DB is rolled back
   before_destroy do
-    raise CannotDestroyIfActive if activated?
+    raise CannotDestroyIfActive if activated? && !data_analyst?
```

Comment:
@mseedat-moj I don't think we want active data analysts to be deletable.

Reply:
This was done to allow rollback, but rollback is now impossible so reverted this rule back